### PR TITLE
Create endpoint to get list of users

### PR DESCRIPTION
**Pull request by AndriiChepeliuk**
--- Add ---

function to get all Users (by IdentityRoleNames.User)
attribute - AuthorizeByRoleAttribute

--- Change ---

constants - IdentityRoleNames to enum
SeedData - profile andrewchepeliuk@gmail.com updated to level "Logist"

### CODE REVIEW

**Review by marynakernychna — DISMISSED** [review]

File: Core/DTO/UserDTO/UserFullInfoDTO.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    public class UserFullInfoDTO
+    {
+        public string Name { get; set; }
+        public string Surname { get; set; }
+        public string UserName { get; set; }
+        public float? Rating { get; set; }
+        public bool HasCar { get; set; }
+        public string Email { get; set; }
+        public bool EmailConfirmed { get; set; }
+        public string PhoneNumber { get; set; }
```
Comment (Grasmus) [inline Core/DTO/UserDTO/UserFullInfoDTO.cs]: We don't use this field. Should it be removed?

Comment (AndriiChepeliuk) [reply Core/DTO/UserDTO/UserFullInfoDTO.cs]: Yes, you are right. Fixed!

**Review by Grasmus — APPROVED** [review]

File: API/Controllers/UserController.cs [hunk 2]
```diff
@@ -3,6 +3,9 @@
 using System.Threading.Tasks;
 using Core.DTO.UserDTO;
 using Microsoft.AspNetCore.Authorization;
+using Core.Constants;
+using System;
```
Comment (marynakernychna) [inline API/Controllers/UserController.cs]: Need to remove the extra using

File: Core/DTO/UserDTO/UserFullInfoDTO.cs [hunk 3]
```diff
@@ -0,0 +1,16 @@
+﻿using System;
+
+namespace Core.DTO.UserDTO
+{
+    public class UserFullInfoDTO
```
Comment (marynakernychna) [inline Core/DTO/UserDTO/UserFullInfoDTO.cs]: Maybe we should rename it "UserDTO"?

File: API/Controllers/UserController.cs (hunk 2 above)
Comment (AndriiChepeliuk) [reply API/Controllers/UserController.cs]: Resolved!

File: Core/DTO/UserDTO/UserFullInfoDTO.cs (hunk 3 above)
Comment (AndriiChepeliuk) [reply Core/DTO/UserDTO/UserFullInfoDTO.cs]: Resolved!

**Review by marynakernychna — APPROVED** [review]

**Pull request merged**